### ISSUE TITLE
Tabs are inaccessible to keyboard and screen reader users

### ISSUE THREAD

**Issue by schne324** [dequelabs/workshop-08-2024#6]
Refer to the ARIA Authoring Practices Guide: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/

- Right Arrow (`key === "ArrowRight"`):
    - Moves focus to the next tab.
    - If focus is on the last tab, moves focus to the first tab.
    - Activates the newly focused tab.
- Left Arrow (`key === "ArrowLeft"`):
    - Moves focus to the previous tab.
    - If focus is on the first tab, moves focus to the last tab.
    - Activates the newly focused tab.

## Dev Notes

The code changes for this fix should be done in `tabs.js` within the `keydown` listener towards the bottom of the file
